Mathlib/Data/Fintype/Card.lean

```diff
@@ -378,14 +378,7 @@ instance (priority := 900) Finite.of_fintype (α : Type*) [Fintype α] : Finite
   Fintype.finite ‹_›
 
 theorem finite_iff_nonempty_fintype (α : Type*) : Finite α ↔ Nonempty (Fintype α) :=
```

Comment:
Would it be sensible to move this theorem too?

Reply:
The `mpr` implication depends on material from this file, so let's leave it here (at least, for now).